Maths/BinomialCoefficient.js

```diff
@@ -0,0 +1,31 @@
+/*
+ * Author: Akshay Dubey (https://github.com/itsAkshayDubey)
+ * Binomial Coefficient: https://en.wikipedia.org/wiki/Binomial_coefficient
+ * function to find binomial coefficient of numbers n and k.
+ * return binomial coefficient of n,k
+ */
+
+/**
+ * @function findBinomialCoefficient
+ * @description -> this function returns bonimial coefficient
+ * of two numbers n & k given by n!/((n-k)!k!)
+ * @param {number} n
+ * @param {number} k
+ * @returns {number}
+ */
+
+import { calcFactorial } from './Factorial'
+
+export const findBinomialCoefficient = (n, k) => {
+  if ((typeof n !== 'number') || (typeof k !== 'number')) {
+    throw Error('Type of arguments must be number.')
+  }
+  if (n <= 0 || k <= 0) {
+    throw Error('Arguments must be greater than zero.')
```

Comment:
`k` can definitely be zero; the formula would also allow for `n` to be zero

Reply:
Fixed in [this](https://github.com/TheAlgorithms/JavaScript/pull/1094/commits/b3c013aa9dc6885b20721cd3c11f239237c8b6e0) commit. Kindly have a look.